helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/InstancesAccessor.java

```diff
@@ -144,7 +144,7 @@ public Response getAllInstances(@PathParam("clusterId") String clusterId,
 
   @POST
   public Response instancesOperations(@PathParam("clusterId") String clusterId,
-      @QueryParam("command") String command, String content) {
+      @QueryParam("skipZKRead") String skipZKRead, @QueryParam("command") String command, String content) {
```

Comment:
Suggest we consolidate the nomenclature for the boolean flag (as explained above).